### PR TITLE
Add number of threads to INFO

### PR DESCRIPTION
**Pull request by y9rabbito**
This is regarding the issue #1904, @royjacobson can you please take your time to see if it's solve the issue or need to make any changes to resolve the issue or in correct track.

### CODE REVIEW

File: src/server/server_family.cc [hunk 1]
```diff
@@ -12,7 +12,7 @@
 #include <absl/strings/strip.h>
 #include <croncpp.h>  // cron::cronexpr
 #include <sys/resource.h>
-
+#include <thread>
```
Comment (romange) [inline src/server/server_family.cc:15]: no need

File: src/server/server_family.cc [hunk 2]
```diff
@@ -1408,6 +1408,10 @@ void ServerFamily::Info(CmdArgList args, ConnectionContext* cntx) {
 
   string info;
 
+  append("number_of_threads", std::thread::hardware_concurrency());
```
Comment (romange) [inline src/server/server_family.cc:1411]: no need

File: src/server/server_family.cc [hunk 3]
```diff
@@ -1408,6 +1408,10 @@ void ServerFamily::Info(CmdArgList args, ConnectionContext* cntx) {
...
+  append("shard_set_size", shard_set->size());
```
Comment (romange) [inline src/server/server_family.cc:1412]: shard_set_size no need, we have now "DEBUG SHARDS"

File: src/server/server_family.cc [hunk 4]
```diff
@@ -1408,6 +1408,10 @@ void ServerFamily::Info(CmdArgList args, ConnectionContext* cntx) {
...
+  append("proactor_pool_size", service_.proactor_pool().size());
```
Comment (romange) [inline src/server/server_family.cc:1413]: please rename it to `thread_count`

Comment (romange) [reply src/server/server_family.cc:1413]: also please move it under "SERVER" section.

File: src/server/server_family.cc (hunk 1 above)
Comment (y9rabbito) [reply src/server/server_family.cc:15]: Thank you @romange i will take care of that and after completing i am commiting the changes

**Comment by y9rabbito** [comment]
Sir @romange could you checked that i have implemented what you have suggest to do me.

File: src/server/server_family.cc [hunk 5]
```diff
@@ -1429,7 +1429,7 @@ void ServerFamily::Info(CmdArgList args, ConnectionContext* cntx) {
   if (should_enter("SERVER")) {
     auto kind = ProactorBase::me()->GetKind();
     const char* multiplex_api = (kind == ProactorBase::IOURING) ? "iouring" : "epoll";
-
+    append("thread_count", service_.proactor_pool().size());
```
Comment (romange) [inline src/server/server_family.cc:1432]: That's good but please move it after tcp_port, thanks 🙂

Comment (y9rabbito) [reply src/server/server_family.cc]: @romange sir i have done that could you please take a look on it.

**Comment by romange** [comment]
@y9rabbito please fix the whitespace issue.

**Comment by y9rabbito** [comment]
> @y9rabbito please fix the whitespace issue.

@romange could you please where is the whitespace issue particulary as I am unable to find it

**Comment by romange** [comment]
@y9rabbito please see https://github.com/dragonflydb/dragonfly/blob/main/CONTRIBUTING.md#before-you-make-your-changes  on how to install precommit checks. Also please see the failing CI checks here: 
https://github.com/dragonflydb/dragonfly/actions/runs/6680938416/job/18156155079?pr=2085

**Comment by y9rabbito** [comment]
> @y9rabbito please see https://github.com/dragonflydb/dragonfly/blob/main/CONTRIBUTING.md#before-you-make-your-changes on how to install precommit checks. Also please see the failing CI checks here: https://github.com/dragonflydb/dragonfly/actions/runs/6680938416/job/18156155079?pr=2085

Sir i try to do with my little knowledge everything works fine but clang formatting errror remains could you please take e look on that @romange

**Review by romange — APPROVED** [review]

**Comment by y9rabbito** [comment]
Thank you sir! @romange

**Pull request merged**